mlos_bench/mlos_bench/config/README.md

```diff
@@ -8,6 +8,13 @@ In general the `config` directory layout follows that of the `mlos_bench` module
 
 Full end-to-end examples are provided in the [`cli`](./cli/) directory, and typically and make use of the root [`CompositeEnvironments`](./environments/root/) to combine multiple [`Environments`](./environments/), also referencing [`Services`](./services/), [`Storage`](./storage/), and [`Optimizer`](./optimizers/) configs, into a single [`mlos_bench`](../run.py) run.
 
+## Config parameters
+
+An `Environment` configuration can have two sections, `const_args` and `tunable_params`.
```

Comment:
Also `required_args`

Reply:
ooh that's right. I will add a section on those in connection with the globals and composite envs